src/Control/Eff/Lift.hs

```diff
@@ -1,34 +1,16 @@
-{-# LANGUAGE DataKinds #-}
-{-# LANGUAGE FlexibleContexts #-}
-{-# LANGUAGE RankNTypes, ScopedTypeVariables #-}
 {-# LANGUAGE Safe #-}
 -- | Lifting primitive Monad types to effectful computations.
 -- We only allow a single Lifted Monad because Monads aren't commutative
 -- (e.g. Maybe (IO a) is functionally distinct from IO (Maybe a)).
 module Control.Eff.Lift ( Lift (..)
-                       , lift
-                       , runLift
-                       , catchDynE
-                       ) where
+                        , lift
+                        , runLift
+                        , catchDynE
+                        ) where
 
-import Control.Eff
+import Control.Eff.Internal
```

Comment:
@greydot any particular reason why we are bringing `Lift` related things from `Eff.Internal`, as opposed to having them be defined in `Eff.Lift` and have `Eff.Lift` depend on `Eff` (i.e., as it was before)?